docs/reference/commandline/ps.md

```diff
@@ -59,7 +99,66 @@ The currently supported filters are:
+    9b6247364a03        busybox                "top"               24 minutes ago      Up 24 minutes                           nostalgic_stallman
+
+This will list all containers that are running.
+
+    $ docker ps --filter status=paused
+    CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS                      PORTS               NAMES
+    673394ef1d4c        busybox             "top"               About an hour ago   Up About an hour (Paused)                       nostalgic_shockley
+
+This will list all containers that are paused.
+
+#### Ancestor
+
+The `ancestor` filter allows to filter containers based on the image or a descendant of it.
+
+    $ docker ps --filter ancestor=ubuntu
+    CONTAINER ID        IMAGE               COMMAND             CREATED              STATUS              PORTS               NAMES
+    919e1179bdb8        ubuntu-c1           "top"               About a minute ago   Up About a minute                       admiring_lovelace
+    5d1e4a540723        ubuntu-c2           "top"               About a minute ago   Up About a minute                       admiring_sammet
+    82a598284012        ubuntu              "top"               3 minutes ago        Up 3 minutes                            sleepy_bose
+    bab2a34ba363        ubuntu              "top"               3 minutes ago        Up 3 minutes                            focused_yonath
+
+
+This will list all containers that are based on the `ubuntu` (`lastest` tag is implicit) or a descendant image of `ubuntu`.
+
+    $ docker ps --filter ancestor=ubuntu-c1
+    CONTAINER ID        IMAGE               COMMAND             CREATED              STATUS              PORTS               NAMES
+    919e1179bdb8        ubuntu-c1           "top"               About a minute ago   Up About a minute                       admiring_lovelace
+
+This will list only the containers that are based on the `ubuntu-c1` image (which, in this case is a child of `ubuntu`).
+
+    $ docker ps --filter ancestor=ubuntu:12.04.5
+    CONTAINER ID        IMAGE               COMMAND             CREATED              STATUS              PORTS               NAMES
+    82a598284012        ubuntu:12.04.5      "top"               3 minutes ago        Up 3 minutes                            sleepy_bose
+
+This will list all containers that are based on the `ubuntu:12.04.5` or a descendant image of `ubuntu:12.04.5`.
+
+    $ docker ps --filter ancestor=d0e008c6cf02
+    CONTAINER ID        IMAGE               COMMAND             CREATED              STATUS              PORTS               NAMES
+    82a598284012        ubuntu:12.04.5      "top"               3 minutes ago        Up 3 minutes                            sleepy_bose
+
+This will list all containers that are based on the layer `d0e008c6cf02` (which is, in this case `ubuntu:12.04.5`) or an image that have this layer in it's layer stack.
```

Comment:
Move prior to actual example.

The following matches iages that are based on the layer `d0e008c6cf02` or an image that have this layer in it's layer stack.